common/celery/routers.py

```diff
@@ -0,0 +1,45 @@
+"""
```

Comment:
can you move this to the `openedx` directory.  The `common` directory is deprecated and we do not add any new files/folders to it.